index.js

```diff
@@ -73,13 +74,20 @@ module.exports = async (Component, opts = {}) => {
   const browser = await puppeteer.launch()
   const page = await browser.newPage()
   await page.goto(data)
+
+  let rect = {}
+  if (!width && !height) {
```

Comment:
The CLI provides default width and height. Might be worth adding an explicit flag for determining dimensions from the element